app/frontend/shared/types/User.res

```diff
@@ -24,10 +24,10 @@ let findById = (id, proxies) =>
   proxies |> ArrayUtils.unsafeFind(proxy => proxy.id == id, "Unable to find a User with ID " ++ id)
 
 let make = (~id, ~name, ~avatarUrl, ~title) => {
-  id: id,
-  name: name,
-  avatarUrl: avatarUrl,
-  title: title,
+  id,
+  name,
+  avatarUrl,
+  title,
```

Comment:
Editor issueIncorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)